collections/environments/PagoPA local.bru

```diff
@@ -18,4 +18,5 @@ vars {
   host-template: http://localhost:3900
   JWT-API-SEC: Bearer {{process.env.JWT-API-SEC}}
   JWT-M2M-ADMIN: Bearer {{process.env.JWT-M2M-ADMIN}}
+  host-in-app-notification: http://localhost:3901
```

Comment:
are we using any convention to choose ports? I see in line 14 they're already using port 4000 but only ports ending in `00` are used throughout